tests/test_message_sqs.py

```diff
@@ -0,0 +1,31 @@
+"""
+ Test script to:
+  - Validate message sent to Skype channel against the message received on SQS
+"""
+import time
+from conf import sqs_conf
```

Comment:
I believe lines 6 and 7 should be combined. But I could be wrong. What does pylint say?